experimental/GaloisGrp/src/Solve.jl

```diff
@@ -295,8 +295,8 @@ julia> G, C = galois_group(x^3-3*x+17)
 
 julia> d = derived_series(G)
 3-element Vector{PermGroup}:
- Sym(3)
- Alt(3)
+ Permutation group of degree 3 and order 6
+ Permutation group of degree 3 and order 3
```

Comment:
I am unhappy about how my experiment with the printing for symmetric and alternating groups works out here. I'd rather go back to how it was... or perhaps this:
```suggestion
 Symmetric group of degree 3 and order 6
 Alternating group of degree 3 and order 3
```